### ISSUE TITLE
Typeface support

### ISSUE THREAD

**Issue by maxkeppeler** [RubenGees/Introduction#11]
- [x] **I have verified that I am using the latest version and searched for exisiting issues** (If this checkbox is not ticked, the issue might be ignored).

I am using version: 1.3  

Is it possible to add the support of changing the typeface of the title and description? I really didn't liked all the other intro libraries. Your's s simply the best in my use case. It works great. Thanks in advance if that would be possible. (The Intro would fit then better to the rest of my app.)

**Comment by rubengees** [comment]
Thank you! :)

I think that should be possible, how about the following additional methods:

`withDescription(String, Typeface)`
`withTitle(String, Typeface)`

Would that fit your needs?

**Comment by maxkeppeler** [comment]
That would be awesome! Absolutely.

**Comment by maxkeppeler** [comment]
One more thing. (Sorry for that). Would it be possible to add methods to specify the text size too? For the title and description?

This and the typeface support is everything I need to change and it would avoid it, to create a custom layout just to change these two little attributes. ':/

**Comment by rubengees** [comment]
Sure, working on it.

**Comment by rubengees** [comment]
Released as [1.3.6](https://github.com/rubengees/introduction/releases/tag/1.3.6).  
The new relevant methods are:

```
IntroductionBuilder#withTypeface(Typeface)
Slide#withTitleSize(Float)
Slide#withDescriptionSize(Float)
```

Please let me know if this works for you.

**Comment by maxkeppeler** [comment]
Thank you very much! I will test it asap.

**Comment by maxkeppeler** [comment]
@rubengees It works absolutely fine! Just one thing, I actually need a typeface per title and description :/ Could you add that too? Until now it's just one global typeface. Thanks in advance!

**Comment by rubengees** [comment]
Ah, sure. Do you need them per slide? This is not possible with the current architecture and I don't want to rewrite everything...
I would go with:
```
IntroductionBuilder#withTitleTypeface(Typeface)
IntroductionBuilder#withDescriptionTypeface(Typeface)
```

Would that work for you?

**Comment by maxkeppeler** [comment]
Sorry for the late response. Exactly that's what I need, thanks!

**Comment by rubengees** [comment]
Great, I'll work at it later.

**Comment by rubengees** [comment]
Sorry for the delay, released as [1.3.7](https://github.com/rubengees/introduction/releases/tag/1.3.7).  
Please let me know if this works for you.

**Comment by rubengees** [comment]
I assume everything works, so I'm closing this. If not, feel free to reopen.
